{"uuid": "7405bb60-783d-4f9f-9a76-5b016f9fc227", "children": ["3581c62d-fa40-4091-8d71-b7dbe17c989b"], "befores": [{"name": "driver", "status": "passed", "start": 1753965241407, "stop": 1753965244852}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#invalidsessionidexception\nStacktrace:\nNoSuchDriverError: A session is either terminated or not started\n    at asyncHandler (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:311:15)\n    at /home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:514:15\n    at Layer.handle [as handle_request] (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/layer.js:95:5)\n    at /home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:284:15\n    at param (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:365:14)\n    at param (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:376:14)\n    at Function.process_params (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:421:3)\n    at next (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:280:10)\n    at logger (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/morgan/index.js:144:5)\n    at Layer.handle [as handle_request] (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:328:13)\n    at /home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:346:12)\n    at next (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:280:10)\n    at jsonParser (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/body-parser/lib/types/json.js:113:7)\n    at Layer.handle [as handle_request] (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:328:13)\n    at /home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/home/ethan/.nvm/versions/node/v16.20.2/lib/node_modules/appium/node_modules/express/lib/router/index.js:346:12)\n", "trace": "  File \"/home/ethan/PycharmProjects/mobile_automation/.venv/lib/python3.10/site-packages/allure_commons/_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/ethan/PycharmProjects/mobile_automation/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n  File \"/home/ethan/PycharmProjects/mobile_automation/tests/conftest.py\", line 8, in driver\n    driver.quit()\n  File \"/home/ethan/PycharmProjects/mobile_automation/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 612, in quit\n    self.execute(Command.QUIT)\n  File \"/home/ethan/PycharmProjects/mobile_automation/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n  File \"/home/ethan/PycharmProjects/mobile_automation/.venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py\", line 125, in check_response\n    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\n"}, "start": 1753965253742, "stop": 1753965253861}], "start": 1753965241407, "stop": 1753965253861}